examples/NextJsAndAzureB2C-master/README.md

```diff
@@ -0,0 +1,157 @@
+# NextJsAndAzureB2C
+
```

Comment:
@sc-addypathania We should maybe at least put the table here from the Readme template.
```suggestion

| | |
|--|--|
| **Framework** | JavaScript Rendering SDK (Next.js) |
| **Sitecore Version** | 10.1 |
| **Author** | Sitecore |

```